journal/original/38-chapter-35.adoc

```diff
@@ -243,7 +243,7 @@ The disciple is not above his master, nor the servant above his lord, etc.
 24th.--Attended the Quarterly Meeting, in which I had to sound an alarm amongst Friends,
 declaring that the Most High will have a church, He will have a people for himself,
 that shall show forth his praise.
-Many were willing and ready to join in the "`Hosannato the son
+Many were willing and ready to join in the "`Hosanna the son
```

Comment:
`Hosanna to` here as well.

Reply:
I must have been too tired working on this last night because I had looked it up in the original and thought the "to" was omitted! Opps!